sn_node_launchpad/src/utils.rs

```diff
@@ -172,17 +144,14 @@ macro_rules! trace_dbg {
 pub fn version() -> String {
     let author = clap::crate_authors!();
 
-    // let current_exe_path = PathBuf::from(clap::crate_name!()).display().to_string();
-    let config_dir_path = get_config_dir().display().to_string();
-    let data_dir_path = get_data_dir().display().to_string();
+    let data_dir_path = get_launchpad_data_dir_path().unwrap().display().to_string();
```

Comment:
no unrwaps please!

Reply:
Hey Josh, the unwrap has to be either used here or inside `#[command(author, version = version(), about)]`. There is no other option, I guess.

Reply:
Regarding the `Clear` widget, It is performed before rendering the popup `f.render_widget(Clear, layer_zero);`

What issues are we seeing there exactly? There are a couple issues when the screen size becomes too small, but that should not be related to this.

Reply:
I saw border lines running through the input field.

Reply:
Does this happen every single time?
I feel like it might be because something else used the stdout and printed stuff there.